demo/scripts/controls/ribbonButtons/tableEditOperations.ts

```diff
@@ -21,19 +24,25 @@ const tableEditOperationsLabel: Record<TableEditOperationsKey, string> = {
     insertBelow: 'Insert Below',
     insertLeft: 'Insert Left',
     insertRight: 'Insert Right',
+    merge: 'Merge Cells',
     deleteTable: 'Delete Table',
     deleteRow: 'Delete Row',
     deleteColumn: 'Delete Column',
+    mergeAbove: 'Merge Above',
+    mergeBelow: 'Merge Below',
```

Comment:
what about merge left and right?

Reply:
I just added this two to test the behavior, I will add the other two.